### PR TITLE
[4][webservices] return tag info on GET content articles

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue # .

### Summary of Changes
return tags on articles webservice get

### Testing Instructions
add some tags to some articles
send a GET to` {{base_url}}/{{base_path}}/content/articles`

### Actual result BEFORE applying this Pull Request

no tag info returned
` "tags": []`
### Expected result AFTER applying this Pull Request

tag info returned
` "tags": {
                    "1": "aTag"
                }`
### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by wilsonge** [comment]
Why are we returning the numeric prefix? That looks incorrect to me.

It needs to be in the relationships of the article - such as the comments section in the main json api examples here https://jsonapi.org/

**Comment by alikon** [comment]
did you see the response of a GET for a single article?

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on ed5da42cf1a43f82436d03708103b1c9a594b9fc<br /><br />Tested successfully in Joomla 4.2.4-dev of 24 October in Wampserver, PHP 8.1.10 and Postman v10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39009">issues.joomla.org/tracker/joomla-cms/39009</a>.</sub>

File: api/components/com_content/src/View/Articles/JsonapiView.php [hunk 1]
```diff
@@ -220,6 +221,14 @@ protected function prepareItem($item)
             $item->tags = array_combine($tagsIds, $tagsNames);
         } else {
             $item->tags = [];
+            $tags = new TagsHelper();
+            $tagsIds = $tags->getTagIds($item->id, 'com_content.article');
+
+            if (!empty($tagsIds)) {
+               $tagsIds    = explode(',',$tagsIds);
```
Comment (Quy) [inline api/components/com_content/src/View/Articles/JsonapiView.php]:
```suggestion
               $tagsIds    = explode(',', $tagsIds);
```

**Comment by richard67** [comment]
I've restored the previous human test result in the issue tracker so it's properly counted because all changes after that were only code style and a clean branch update.

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 6b03dcab8f6840bb632b19fa68881444f44d4ddd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39009">issues.joomla.org/tracker/joomla-cms/39009</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39009">issues.joomla.org/tracker/joomla-cms/39009</a>.</sub>

**Comment by fancyFranci** [comment]
Thank you for your improvement.

**Pull request merged**